frontend/simple/views/components/i18n.vue

```diff
@@ -12,7 +12,7 @@ export default {
       default: 'span'
     }
   },
-  computed: {
+  methods: {
```

Comment:
(I've already wrote this, but the comment disappeared, so I'm writing it again, sorry if you get a double notification)

@taoeffect there was a weird bug on input save/delete buttons when using i18n.

![sep-05-2018 21-20-43](https://user-images.githubusercontent.com/14869087/45123251-393ad800-b15e-11e8-99a4-9e26532a56ac.gif)

Note how the delete button (red color) has the Save button text "save" and not "delete". The DOM/JS is correct: The delete button (red save), deletes the contribution and the save button (blue save), updates the text.

This was really tricky and I took a while to understand the problem was from i18n and not the component itself. Changing `translatedText` from `computed` to `methods` seemed to fix the issue, but honestly I can't explain exactly why that bug happened...